src/main/java/hu/wurfel/refference/school/mark/MarkService.java

```diff
@@ -129,72 +139,72 @@ private void mWTeacher(MarkRequestForSearch markRequestForSearch) {
         }
     }
 
-    protected ArrayList<Mark> getMarksByDiary(@NotNull Diary diary) {
+    protected List<Mark> getMarksByDiary(@NotNull Diary diary) {
         return getMarksByDiaryid(diary.getId());
     }
 
-    protected ArrayList<Mark> getMarksByDiaries(@NotNull ArrayList<Diary> diaries) {
-        ArrayList<Mark> marks = new ArrayList<>();
+    protected List<Mark> getMarksByDiaries(@NotNull List<Diary> diaries) {
+        List<Mark> marks = new ArrayList<>();
         for (Diary diary : diaries) {
             marks.addAll(getMarksByDiary(diary));
         }
         return marks;
     }
 
-    protected ArrayList<Mark> getMarksByStudent(@NotNull Student student) {
-        ArrayList<Mark> marks = new ArrayList<>();
-        for (Diary diary : new ArrayList<Diary>(diaryCrudService.getDiariesByScid(student.getId()))) {
+    protected List<Mark> getMarksByStudent(@NotNull Student student) {
+        List<Mark> marks = new ArrayList<>();
+        for (Diary diary : new ArrayList<>(diaryCrudService.getDiariesByScid(student.getId()))) {
```

Comment:
Első körben wtf is `Scid`? Még ha tudom is éppen pont, hogy mit akar jelenteni, ha bárkinek találgatnia kell, hogy vajh' mit jelenthet egy elnevezés, akkor az egy rossz elnevezés.

Reply:
minden hasonló átnevezve